tests/unit/tensor.cpp

```diff
@@ -2096,3 +2096,36 @@ TEST_F(NegativeTensor, MutableDescriptors) {
     vk::CreateDescriptorSetLayout(device(), &ds_layout_ci, NULL, &ds_layout);
     m_errorMonitor->VerifyFound();
 }
+
+TEST_F(NegativeTensor, DescriptorBindingUpdateAfterBindTensorNoFeature) {
+    TEST_DESCRIPTION("Call UpdateAfterBind on tensor but feature is not enabled.");
+
+    SetTargetApiVersion(VK_API_VERSION_1_4);
+    AddRequiredExtensions(VK_ARM_TENSORS_EXTENSION_NAME);
+    AddRequiredFeature(vkt::Feature::tensors);
+    // intentionally missing: Feature::descriptorBindingStorageTensorUpdateAfterBind
+    RETURN_IF_SKIP(Init());
+
+    VkDescriptorSetLayoutBinding binding{};
```

Comment:
nit: We want it in this form, easier to morph into the Descriptor helpers later

```suggestion
    VkDescriptorSetLayoutBinding binding{0, VK_DESCRIPTOR_TYPE_TENSOR_ARM, 1, VK_SHADER_STAGE_ALL, nullptr};
```